MahApps.Metro/Styles/Controls.DataGrid.xaml

```diff
@@ -250,6 +250,10 @@
 
     <Style x:Key="MetroDataGridRowHeader"
            TargetType="{x:Type DataGridRowHeader}">
+        <Setter Property="HorizontalContentAlignment"
+                Value="Left" />
```

Comment:
yeah i know, but the default before your changes is Stretch (it's the default if you doesn't say HorizontalAlignement="xyz")